resources/shaders/rt_hybrid.hlsl

```diff
@@ -259,7 +301,7 @@ void ReflectionHit(inout ReflectionHitInfo payload, in MyAttributes attr)
 	payload.cone = Propagate(payload.cone, 0, length(payload.origin - hit_pos));
 
 	// Calculate the texture LOD level 
-	float mip_level = ComputeTextureLOD(
+	float mip_level = 0; /*ComputeTextureLOD(			Mipmapping
```

Comment:
Commented out block of code? Leftover debug code or can it be deleted?

Reply:
It can't be deleted, it's there because of https://github.com/TeamWisp/WispRenderer/issues/284. Mipmapping does work with perfect reflections, but with variable roughness can sample from the incorrect mip at some angles. I don't want to remove it but I don't want to keep it in either.